api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java

```diff
@@ -93,16 +93,13 @@ private void installConcepts() {
         	Context.flushSession(); //Flush so that purges are not deferred until after data import
         	
         	DataImporter dataImporter = Context.getRegisteredComponent("dataImporter", DataImporter.class);
-            dataImporter.importData("Reference_Application_Concepts-25.xml");
-            dataImporter.importData("Covid-19_Concepts-1.xml");
+            dataImporter.importData("Reference_Application_Concepts-24.xml");     
```

Comment:
did you just rename this `Reference_Application_Concepts-25.xml`   to `Reference_Application_Concepts-24.xml`   ??
are you sure we have a package named `Reference_Application_Concepts-24.xml`  ??

Reply:
Sorry for that mistake, i have resolved